TRANSMUTANSTEIN/AuthResponseTest.cs

```
@@ -145,8 +144,8 @@ public void TestAuthResponseConstruction()
         Assert.AreEqual(accountDetails.AccountName, authResponse.Nickname);
         Assert.AreEqual(accountDetails.Email, authResponse.Email);
         Assert.AreEqual(clientIpAddress, authResponse.ClientIpAddress);
-        Assert.AreEqual(cookie, authResponse.Cookie);
-        Assert.AreEqual("82d5a30da745ff372d17434b683a5e9b2db8f0ea", authResponse.AuthHash);
+        Assert.AreEqual(accountDetails.Cookie, authResponse.Cookie);
+        Assert.AreEqual(AuthResponse.ComputeAuthHash(accountDetails.AccountId, accountDetails.Cookie, clientIpAddress), authResponse.AuthHash);

```

For line 148, I actually like having the hardcoded value. Otherwise this test could fail due to bugs in `ComputeAuthHash` which is independent of the code under test (or at best is an implementation detail).

It also makes the test easier to read, as you know exactly what the expected output should be rather than doing additional logic of a function call.

---

This is coming from a diffbase. I'll address this in a separate PR.